core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -284,26 +288,18 @@ If the component is used on a background with a color of grey-50, mode-variant p
 
 ### Updated `<sdds-inline-tabs>` and `<sdds-navigation-tabs>` and introduced `<sdds-folder-tabs>`. Introduced child components for all Tabs.
 
-Previously we had the `<sdds-inline-tabs>` and `<sdds-inline-tabs-fullbleed>`. These
-reminded little of eachother in appearence and therefore it made more sense to treat
-these as two totally seperate component. To have a clear distiction between them we
-have now renamed the `<ssd-inline-tabs>` to `<sdds-folder-tabs>`, to futher highlight
-its look and its function.
+Previously we had the `<sdds-inline-tabs>` and `<sdds-inline-tabs-fullbleed>`. These had little resemblance to each other in appearance, thus it was more logical to consider them as completely distinct components. 
+In order to establish a clear differentiation between them, we have recently renamed the `<ssd-inline-tabs>` as `<sdds-folder-tabs>` to better emphasize both its visual appearance and its functionality.
```

Comment:
```suggestion
In order to establish a clear differentiation between them, we have recently renamed the `<ssd-inline-tabs>` as `<sdds-folder-tabs>` to better reflect its visual appearance.
```
I think the use of "folder" in the name has no connection to the functionality so I'd leave that out.

Reply:
Also, your comment highlighted the fact that it was misspelled also, <ssd-inline-tabs> is not correct :).